lib/w_common.dart

```diff
@@ -12,7 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export 'disposable.dart' show Disposable;
+export 'disposable_vm.dart'
```

Comment:
I would suggest just exporting `disposable_vm.dart` without the `show` clause. Since `disposable_vm.dart` is already filtering what is being exported, we won't need to remember to keep the list here in sync with the list there in the future.